src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const result = sourceString
+    .split(';')
+    .map((item) => item.split(':'))
+    .map((item) => item.map((value) => value.trim()))
+    .map((item) => item.filter((value) => value !== ''))
+    .filter((item) => item.length !== 0)
+    .reduce((object, item) => {
+      object[item[0]] = item[1];
+
+      return object;
+    }, {});
+
+  return result;
```

Comment:
While the chaining of array methods is correct, the indentation should be consistent to improve readability. Each chained method should start on a new line with the same indentation level.